Dockerfile.apk

```diff
@@ -30,11 +30,14 @@ RUN set -ex; \
     fi \
     && tar -C / -xzf /tmp/kong.apk.tar.gz \
     && apk add --no-cache libstdc++ libgcc perl tzdata libcap zlib zlib-dev bash \
-    && adduser -S kong \
     && addgroup -S kong \
+    && adduser -S -G kong kong \
     && mkdir -p "${KONG_PREFIX}" \
-    && chown -R kong:0 ${KONG_PREFIX} \
-    && chown kong:0 /usr/local/bin/kong \
+    && chown -R root:kong ${KONG_PREFIX} \
```

Comment:
@Tieske Is there more context on the motivation for this change? IIRC, the owner of kong files was changed to `kong` fairly recently, so I am afraid of changing it back to root.

Reply:
In the original pr by Colin; Changing the ownership to root but giving the Kong user access prevents the Kong user from modifying the files.

Reply:
generally, everything under a common shared path like `/usr/local/lib`, `/usr/local/share` should always be owned by `root:root` because technically other packages will also write to it. this applies to also /etc/kong/kong.conf; only
/usr/local/kong (and /usr/local/openresty, if we mark our package a conflict with `openresty`) should be owned by
`kong:kong`.

Reply:
The course of action is IMO:
- Consolidate what the permissions should be (I tend to agree with most of what @fffonion said)
- Move permission-related code to post-install scripts (alpine is an exception since it's not really a distro package) as suggested by Wangchong. I cannot remember of any particular issues around this explaining why they were placed in the docker images only.